Cloud Pak for Data/WML/notebooks/model risk management/OpenScale MRM Cloud pak for Data.ipynb

```diff
@@ -64,12 +64,13 @@
    "source": [
     "!pip install --upgrade pyspark==3.0.3 --no-cache | tail -n 1\n",
```

Comment:
We should install pyspark 3.2.x because that's what is given below spark-mllib_3.2